src/main/java/org/myrobotlab/service/Log.java

```diff
@@ -254,6 +254,10 @@ public void setRootLogLevel(String level) {
     getLogLevel();
     broadcastState();
   }
+  
+  @Override
+  public void setName(String name) {    

```

Comment:
why is this here?

Reply:
required when implementing a Logger interface - ask jvm and jlog makers